tools/settings.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+/**
+ * Outputs a JSON object representing the extension settings based on a settings
+ * file and the package environment.
+ */
+
+'use strict';
+
+const path = require('path');
+const gitDescribeSync = require('git-describe').gitDescribeSync;
+
+// Suppress (expected) EPIPE errors on STDOUT
```

Comment:
Oh and if it's not clear calling `tools/settings.js <settings_file> | cmp -s ...` in the Makefile results in this scenario.